ci/jenkins/Jenkins_steps.groovy

```diff
@@ -80,278 +80,294 @@ def python3_gpu_ut_cython(docker_container_name) {
 
 //------------------------------------------------------------------------------------------
 
-def compile_unix_cpu_openblas() {
+def compile_unix_cpu_openblas(lib_name) {
```

Comment:
Why make this a parameter? In its current design, lib_name depends on the actual compilation. So it's not interchangeable and you can't just call the same function with different lib_name. 

In a bigger picture, most lib_name should be deleted, consolidated to a single one based on having a single directory in which the build places all artifacts (install target)